### PR TITLE
added new activation email

### PR DESCRIPTION
**Pull request by YuYing-Liang**
## Overview

- Added new formatting and styling to our account activation email

## Unit Tests Created

- same as usual

## Steps to QA

- in hackathon_site/settings/__init__.py change the REGISTRATION_OPEN_DATE to a date in the past
- Register for the hackathon by creating an account, the information you enter is not important

- you should see an "activate your account" message on the success page
- in development mode the email won't actually send, it'll store an SMTP log in the hackathon_site/emails directory
- you should see a new log file appear in that directory (it is gitignored so it might be greyed out)
- change the extension of the log file to an html file and open it in your browser (you can open the file in file explorer and open it in the browser from there)
- you should see something like this:

image

repeated twice (but don't worry about it repeating twice that's normal for SMTP)
**Please make sure all the links and buttons (even the ones in the footer) work correctly**
**Check that the email makes sense and there are no grammatical errors**

Any feedback about the email layout/styling is appreciated :)

### CODE REVIEW

**Review by lukecheseldine — APPROVED** [review]
looks really good yuying! all the buttons work and i don't see any grammar mistakes :)

i was a little confused when i clicked the verify email button though because it brought me straight to the log in page, and i wasn't sure if my account had actually been verified, or if the link was wrong. it might be a good idea to display some kind of banner or alert when users get rerouted to the log in so that they know the verification worked.

i noticed too you don't actually link to the NewHacks landing page anywhere on the email. it might be nice if the logos linked to the landing, or if the NewHacks text in the copyright was a link.

those are minor things though and i think the design looks great, so i think it's good to go!

File: .gitignore [hunk 1]
```diff
@@ -145,6 +145,7 @@ cython_debug/
 # Collected static files and media files
 hackathon_site/static/
 hackathon_site/media/
+hackathon_site/emails/
```
Comment (Leo6Leo) [inline .gitignore:148]: we don't want to upload our email template?

**Review by Leo6Leo — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (YuYing-Liang) [reply .gitignore:148]: This directory is generated by Django's email backend for development purposes https://docs.djangoproject.com/en/4.1/topics/email/#topic-email-file-backend so we wouldn't want this in the repo bc the contents would be different for everyone. Similar to how uploading resumes work.

**Comment by YuYing-Liang** [comment]
@lukecheseldine due to the urgency of this ticket I'll merge it first since the changes you suggested are not blocking, i've created a new ticket to address them later: https://ieeeuoft.atlassian.net/browse/NH-18

**Pull request merged**